frontend/src/components/chat/ChatInterface.tsx

```diff
@@ -95,9 +102,12 @@ function ChatInterface() {
       </div>
       <div className="flex-1 flex flex-col relative min-h-0">
         <div
+          data-testid="region"
           ref={scrollRef}
           className="overflow-y-auto p-3"
           onScroll={(e) => onChatBodyScroll(e.currentTarget)}
+          role="region"
```

Comment:
This may cause more confusion that setting no role: See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role

> The region role should be reserved for sections of content sufficiently important that users will likely want to navigate to the section easily and to have it listed in a summary of the page.

> should only be used if the section needing to be identified is not accurately described by one of the other landmark roles

> Every element with a region role should include a label that describes the purpose of the content in the region

Reply:
Oh, ok. I'll revert that then. 😃